pytest_django/plugin.py

```diff
@@ -608,12 +609,13 @@ def mailoutbox(
     _dj_autoclear_mailbox: None,
 ) -> list[django.core.mail.EmailMessage] | None:
     """A clean email outbox to which Django-generated emails are sent."""
-    if not django_settings_is_configured():
-        return None
+    skip_if_no_django()
 
     from django.core import mail
 
-    return mail.outbox  # type: ignore[no-any-return]
+    if hasattr(mail, "outbox"):
+        return mail.outbox  # type: ignore[no-any-return]
+    return None
```

Comment:
I'm not sure about returning `None`. The situation is that someone explicitly used the `mailoutbox` fixture, but didn't properly configure Django to actually fill it. Do they expect `None`? I think probably not.

So I think we should error in this case, with a proper message telling the user what they need to do to get the `mailoutbox` fixture to work (configure email backend to locmem), or if an error is too harsh, then a warning and return `[]`, this way we at least get rid of the pesky `None` that no one expects. I'd go with the error.

Reply:
I was on the fence about this one..